chapter-06/recipe-02/abstract.md

```diff
@@ -0,0 +1,15 @@
+In this recipe, we generate `print_info.c` from the template `print_info.c.in`
```

Comment:
:+1: For starting to write abstracts for recipes!

Reply:
I will add more soon but here we needed one urgently.